_posts/2021-09-23-hacktoberfest.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: Hacktoberfest 2021
+layout: post
+---
+
+Are you ready for Hacktoberfest 2021?  Void Linux is!  We're
+excited to be participating for our 5th year.  Contributions that help to
+address our out-of-date packages queue are especially welcome.
+These are all great ways to dip your feet into the world of Linux
+distro package management and what happens behind the scenes to 
+provide a wide selection of packages and make sure your system remains up to date.
+
+Updating packages is very easy.  You can select a package from the
+list of [out of date
+packages](http://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt)
+and update it using the tools in the [void-packages
+repo](https://github.com/void-linux/void-packages).  The
+[manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+might be of assistance when you are updating packages.
+
+As a general rule, we would prefer Hacktoberfest contributors that do
```

Comment:
Can't figure out how to select multiple lines from this phone, recommend:

> As a general rule, we prefer that newcomers to Void steer clear of "structural" [...]

Of course, newcomers probably don't really know what a "structural package is...